nano/core_test/network.cpp

```diff
@@ -2134,9 +2164,62 @@ TEST (confirmation_height, observers)
 		confirmed = node1->ledger.block_confirmed (transaction, send1->hash ());
 		ASSERT_NO_ERROR (system.poll ());
 	}
+	ASSERT_EQ (1, node1->stats.count (nano::stat::type::confirmation_height, nano::stat::detail::blocks_confirmed, nano::stat::dir::in));
 	ASSERT_EQ (1, node1->stats.count (nano::stat::type::http_callback, nano::stat::detail::http_callback, nano::stat::dir::out));
 }
 
+namespace nano
+{
+TEST (confirmation_height, pending_observer_callbacks)
+{
+	nano::system system;
+	nano::node_flags node_flags;
+	node_flags.delay_frontier_confirmation_height_updating = true;
```

Comment:
This has been added to quite a few tests. For another PR, but I wonder if this should be default true if testnet, then set it to false in `node.bootstrap_confirm_frontiers`

Reply:
Yeh probably :)